host/4/bookworm/java/java21/java21-appservice.Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG JAVA_VERSION=21.0.1
 ARG JAVA_HOME=/usr/lib/jvm/msft-21-x64
 # Build the runtime from source
-ARG HOST_VERSION=4.34.2
+ARG HOST_VERSION=4.1035.0
```

Comment:
The final stage (line 43) still appears to be using .NET 6. Shouldn't it be using .NET 8.0 instead?

Reply:
Could you please use 4.1035.1 ?